framework/src/PhaseSpaceBoundary.cpp

```diff
@@ -587,7 +587,7 @@ unsigned int PhaseSpaceBoundary::GetDiscreteIndex( DataPoint* Input, const bool
 		cerr << "This is a SERIOUS MISCONFIGURATION!!! Exiting :(" << endl;
 		Input->Print();
 		this->Print();
-		DebugClass::SegFault();
```

Comment:
could you replace the DebugClass method with SIGSEGV then? (The compilers have improved recently, a load of these never used to get picked up)